examples/service-notebook/external/jupyterhub_config.py

```diff
@@ -1,15 +1,33 @@
 # our user list
```

Comment:
`examples/service-notebook/managed` content needs to be updated the same way. I have changed that content on my local env and it was working fine.